management/src/test/resources/ehcache-management-4.xml

```diff
@@ -0,0 +1,29 @@
+<config
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns='http://www.ehcache.org/v3'
+    xmlns:mnm='http://www.ehcache.org/v3/management'
+    xsi:schemaLocation="http://www.ehcache.org/v3 ../../../../xml/src/main/resources/ehcache-core.xsd
+                        http://www.ehcache.org/v3/management ../../main/resources/org/ehcache/management/ehcache-management-ext.xsd">
+
+  <service>
+    <mnm:management cacheManagerAlias="my-cache-manager-name">
+      <mnm:tags>
+        <mnm:tag>webapp-name</mnm:tag>
+        <mnm:tag>jboss-1</mnm:tag>
+        <mnm:tag>server-node-1</mnm:tag>
+      </mnm:tags>
+      <mnm:statistics>
+        <mnm:configuration averageWindowDuration="2"
```

Comment:
`minutes`.
If you look at the sample 3, all the fields are there.

I declared some default in the XSD...

I hesitated between using optional fields with default values or required fields, but you need to set all.

What do you think ?